modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/internal/upgrade/registry/ListTypeServiceUpgradeStepRegistrator.java

```diff
@@ -40,6 +40,17 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 				}
 
 			});
+
+		registry.register(
+			"1.1.0", "1.2.0",
+			new BaseExternalReferenceCodeUpgradeProcess() {
+
+				@Override
+				protected String[][] getTableAndPrimaryKeyColumnNames() {
+					return new String[][] {{"ListTypeEntry", "key_"}};
```

Comment:
Is it working? it seems that this base class just works for primary key columns, we should create our own upgrade logic, please see this [example](https://github.com/brianchandotcom/liferay-portal/commit/421c02a7e9606524c834f1e0aca2fe0a001a320e#diff-c0c8d933cfebc440367c9eacb6e53c9d7c9e30980d0dff5367e9b366e3e13898)